pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -414,6 +414,10 @@ func (asw *actualStateOfWorld) IsVolumeReconstructed(volumeName v1.UniqueVolumeN
 	if !ok {
 		return false
 	}
+
+	if podName == operationexecutor.EmptyUniquePodName {
```

Comment:
what is reason for this one? and how it is different from the logic in line 407?

Reply:
So if pod name is empty and volume exists in `foundDuringReconstruction` map, then there is no need to match for pod name, because we are looking for just unique volume name in global path and not any pod specific path.

Reply:
Please turn this explanation into a comment.

Reply:
Added a new function to check if device was reconstructed.